project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
+
+Current squad boards:
+
+- [Snappy squad tasks](https://app.zenhub.com/workspace/o/ubuntudesign/snappy-squad-tasks/boards)
+- [Web squad](https://app.zenhub.com/workspace/o/ubuntudesign/web-squad/boards)
+- [MAAS design squad](https://app.zenhub.com/workspace/o/ubuntudesign/maas-design-squad/boards)
+- [Juju squad](https://app.zenhub.com/workspace/o/ubuntudesign/juju-squad/boards)
+- [Vanilla squad](https://app.zenhub.com/workspace/o/ubuntudesign/vanilla-squad/boards)
+
+## 1. Iterations
+
+- Each iteration should be a milestone in ZenHub in the format `Iteration [Iteration number]-[enddate].[endmonth]`.
+
+ For example `Iteration 9-20.10` followed by `Iteration 10-3.11`.
+
+## 2. Epics
+
+- Each milestone (iteration) should include all epics to be worked on.
+- Epics should have everyone who will be working on it assigned.
+- Epics should always be in the `Iteration Epics` pipeline (see 4.3).
+- Epics should have a description explaining the goal.
+
+## 3. Issues
+
+- Issues for the current milestone should start in the `Current Iteration` pipeline (see 4.4).
+- There should be only one issues per person in the `In Progress` pipeline (see 4.5).
+- Issues should only have one assignee at any one time. If an issue requires multiple people at different stages it should be handed over and the assignee changed at the appropriate time. When reassigned, issues should be moved back into the `Current Iteration` pipeline.
+- Issues should have an estimate based on [the fibonacci estimation guide](https://github.com/canonical-webteam/practices/blob/master/project-management/fibonacci-estimation-guide.md).
+- If for some reason you move to a different issue before completing your current issue, that issue should be moved in to either the `Current Iteration` or `Blocked` pipelines.
+- Issues should have a description if the title isn't obvious.
+- Issues do not need to be part of an epic, but most should.
```

Comment:
This will cause the ZenHub burn down chart to be different from the backlog sheet.

Reply:
My thinking behind this was that working on Juju I pick up small bugs etc. but I don't create a new squad board issue, I just use the issue from juju-gui, as such I don't create an epic. I agree it messes with burn down so maybe there should be some wording around how to deal with the situation I described?